modules/telemetry-showing-data-collected-from-the-cluster.adoc

```diff
@@ -183,5 +183,6 @@ openshift-monitoring -o jsonpath="{.spec.host}")/federate \
 --data-urlencode 'match[]={__name__="namespace:noobaa_usage:max"}' \
 --data-urlencode 'match[]={__name__="namespace:noobaa_system_health_status:max"}' \
 --data-urlencode 'match[]={__name__="ocs_advanced_feature_usage"}' \
---data-urlencode 'match[]={__name__="os_image_url_override:sum"}'
+--data-urlencode 'match[]={_name_="os_image_url_override:sum"}' \
+--data-urlencode 'match[]={_name_="openshift:openshift_network_operator_ipsec_state:info"}'
```

Comment:
{__name__="openshift:openshift_network_operator_ipsec_state:info"}, it seems `__name__`, not `_name_`
@JoshSalomon  can you help confirm?

Reply:
@huiran0826, so this works:

```
curl -G -k -H "Authorization: Bearer $(oc whoami -t)" \
                  https://$(oc get route prometheus-k8s-federate -n \
              openshift-monitoring -o jsonpath="{.spec.host}")/federate \
                  --data-urlencode 'match[]={__name__="openshift:openshift_network_operator_ipsec_state:info"}'
# TYPE openshift:openshift_network_operator_ipsec_state:info untyped
openshift:openshift_network_operator_ipsec_state:info{is_legacy_api="false",mode="Disabled",instance="",prometheus="openshift-monitoring/k8s",prometheus_replica="prometheus-k8s-1"} 1 1719242955034
```